src/iop/filmicrgb.c

```diff
@@ -3913,15 +3910,17 @@ void gui_init(dt_iop_module_t *self)
                                                     "useful to give a safety margin to extreme luminances."));
 
   // Auto tune slider
-  g->auto_button = dt_color_picker_new(self, DT_COLOR_PICKER_AREA, dt_bauhaus_combobox_new(self));
-  dt_bauhaus_widget_set_label(g->auto_button, NULL, N_("auto tune levels"));
```

Comment:
this will remove shortcuts

Reply:
Ok, but I have no idea how to update those lines to my code. Will need some help here.

Reply:
@dterrahe: tested on master and even if I can assign a shortcut here (and not indeed with my PR), shortcut can't be used after. On master, for that icon, when using shortcut (filmic has the focus), I have this error:

![image](https://user-images.githubusercontent.com/45535283/159371679-eefb2e3e-6ed5-4b9f-9d60-502014b4d6cf.png)

And nothing happen, no auto-tune. I've tested with different shortcuts, same error.

Reply:
if I understand well the code, the shortcuts on bauhaus combobox are not related to the quad toggle, but to the main part of the widget (@dterrahe : correct me if I'm wrong)
So I think that it's expected that shortcuts does nothing (before this PR) as the combobox is a "false" one (without any item in the list of choices)

To implement the shortcut here, I would say that `dt_color_picker_new` should use `dt_iop_togglebutton_new` instead of "classic" `dtgtk_togglebutton_new`
But I fear of the consequences, due to how much `dt_color_picker_new` is used...
@dterrahe : what is your opinion on this ?

Reply:
If you select the "button" element for the shortcut (or use visual mapping) the shortcut should work.

Reply:
@dterrahe: last commit added fix shortcut. Thanks to @AlicVB work.